docs/Cli.md

```diff
@@ -200,14 +200,14 @@ Click on a variable to jump to the relevant documentation page.
 | `imu_dcm_ki`                                  | Inertial Measurement Unit KI Gain                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | 0      | 20000  | 0                | Master       | UINT16   |
 | `alt_hold_deadband`                           | Altitude will be held when throttle is centered with an error margin defined in this parameter.                                                                                                                                                                                                                                                                                                                                                                                                                          | 1      | 250    | 40               | Profile      | UINT8    |
-| `alt_hold_fast_change`                        | Authorise fast altitude changes. Should be disabled when slow changes are prefered, for example for aerial photography.                                                                                                                                                                                                                                                                                                                                                                                                  | OFF    | ON     | ON               | Profile      | UINT8    |
+| `alt_hold_fast_change`                        | Authorize fast altitude changes. Should be disabled when slow changes are preferred, for example for aerial photography.                                                                                                                                                                                                                                                                                                                                                                                                  | OFF    | ON     | ON               | Profile      | UINT8    |
```

Comment:
"Authorise" is also correct.

Reply:
Disagree ;-)

Reply:
https://english.stackexchange.com/questions/282621/authorization-vs-authorisation-im-in-some-real-dilemma

Reply:
I'll revert this change. If there's no clear improvement in readability, the change is not useful.